bentoml/fastai.py

```diff
@@ -1,7 +1,48 @@
-raise NotImplementedError(
-    f"""\
-Support for "{__name__}" is temporarily unavailable as BentoML transition to the new \
-design in version 1.0.0 release. Before this module is officially implemented in \
-BentoML, users may use Custom Runner as a workaround. Learn more at http://docs.bentoml.org
-"""
-)
+from __future__ import annotations
+
+import logging
+from typing import Any
+from typing import TYPE_CHECKING
+
+from ._internal.frameworks.fastai import get
+from ._internal.frameworks.fastai import load_model
+from ._internal.frameworks.fastai import save_model
+from ._internal.frameworks.fastai import get_runnable
+
+if TYPE_CHECKING:
+
+    from .models import Model
+    from ._internal.tag import Tag
+    from ._internal.runner import Runner
+    from ._internal.frameworks.fastai import Learner
+
+logger = logging.getLogger(__name__)
+
+
+def save(tag: str, *args: Any, **kwargs: Any) -> Tag:
+    logger.warning(
+        f'The "{__name__}.save" method is being deprecated. Use "{__name__}.save_model" instead'
+    )
+    return save_model(tag, *args, **kwargs)
+
+
+def load(tag: str | Tag | Model, *args: Any, **kwargs: Any) -> Learner:
+    logger.warning(
+        f'The "{__name__}.load" method is being deprecated. Use "{__name__}.load_model" instead'
+    )
+    return load_model(tag, *args, **kwargs)
```

Comment:
This is unrelated to this PR, but shouldn't this be doing `get` instead?

Reply:
i don't think so we still want to load the model right?

Reply:
Nevermind, I think I remembered what this did wrong.